tests/bluetooth/tester/src/btp_vcp.c

```diff
@@ -0,0 +1,544 @@
+	int err;
+
+	/* Default values */
+	vcp_register_param.step = 1;
+	vcp_register_param.mute = BT_VCP_STATE_UNMUTED;
+	vcp_register_param.volume = 100;
+	vcp_register_param.cb = &vcs_cb;
+
+	LOG_DBG("");
+
+	err = bt_vcp_vol_rend_register(&vcp_register_param);
+	if (err) {
+		tester_rsp(BTP_SERVICE_ID_VCS, VCS_INIT, CONTROLLER_INDEX,
+			   BTP_STATUS_FAILED);
+	}
+}
+
+void tester_handle_vcs(uint8_t opcode, uint8_t index, uint8_t *data,
+		       uint16_t len)
+{
+	switch (opcode) {
+	case VCS_READ_SUPPORTED_COMMANDS:
+		vcs_supported_commands(data, len);
+		break;
+	case VCS_INIT:
+		vcp_init();
+		break;
+	case VCS_SET_VOL:
+		set_volume(data, len);
+		break;
+	case VCS_VOL_UP:
+		vol_up();
+		break;
+	case VCS_VOL_DOWN:
+		vol_down();
+		break;
+	case VCS_MUTE:
+		mute();
+		break;
+	case VCS_UNMUTE:
+		unmute();
+		break;
+	default:
+		tester_rsp(BTP_SERVICE_ID_VCS, opcode, index,
+			   BTP_STATUS_UNKNOWN_CMD);
+		break;
+	}
+}
+
+/* Audio Input Control Service */
+static void aics_supported_commands(uint8_t *data, uint16_t len)
+{
+	struct net_buf_simple *buf = NET_BUF_SIMPLE(BTP_DATA_MAX_SIZE);
+
+	net_buf_simple_init(buf, 0);
+	net_buf_simple_add_u8(buf, AICS_READ_SUPPORTED_COMMANDS);
+	net_buf_simple_add_u8(buf, AICS_INIT);
+	net_buf_simple_add_u8(buf, AICS_SET_GAIN);
+	net_buf_simple_add_u8(buf, AICS_MUTE);
+	net_buf_simple_add_u8(buf, AICS_UNMUTE);
+	net_buf_simple_add_u8(buf, AICS_MAN_GAIN);
+	net_buf_simple_add_u8(buf, AICS_AUTO_GAIN);
+	net_buf_simple_add_u8(buf, AICS_MAN_GAIN_ONLY);
+	net_buf_simple_add_u8(buf, AICS_AUTO_GAIN_ONLY);
+	net_buf_simple_add_u8(buf, AICS_DESCRIPTION);
+
+	tester_send(BTP_SERVICE_ID_AICS, AICS_READ_SUPPORTED_COMMANDS,
+		    CONTROLLER_INDEX, buf->data, buf->len);
+}
+
+static void aics_state_cb(struct bt_aics *inst, int err, int8_t gain,
+			  uint8_t mute, uint8_t mode)
+{
+	LOG_DBG("AICS state callback");
+}
+
+static void aics_gain_setting_cb(struct bt_aics *inst, int err, uint8_t units,
+				 int8_t minimum, int8_t maximum)
+{
+	LOG_DBG("AICS gain setting callback");
+}
+
+static void aics_input_type_cb(struct bt_aics *inst, int err,
+			       uint8_t input_type)
+{
+	LOG_DBG("AICS input type callback");
+}
+
+static void aics_status_cb(struct bt_aics *inst, int err, bool active)
+{
+	LOG_DBG("AICS status callback");
+}
+
+static void aics_description_cb(struct bt_aics *inst, int err,
+				char *description)
+{
+	LOG_DBG("AICS description callback");
+}
+
+static struct bt_aics_cb aics_cb = {
+	.state = aics_state_cb,
+	.gain_setting = aics_gain_setting_cb,
+	.type = aics_input_type_cb,
+	.status = aics_status_cb,
+	.description = aics_description_cb
+};
+
+void aics_set_gain(uint8_t *data, uint8_t len)
+{
+	const struct aics_set_gain *cmd = (void *) data;
+	uint32_t gain;
+	int err;
+
+	gain = sys_le32_to_cpu(cmd->gain);
+
+	LOG_DBG("AICS set gain 0x%04x", gain);
+
+	for (int i = 0; i < (int)CONFIG_BT_VCP_VOL_REND_AICS_INSTANCE_COUNT; i++) {
```

Comment:
```suggestion
	for (int i = 0; i < CONFIG_BT_VCP_VOL_REND_AICS_INSTANCE_COUNT; i++) {
```

The Kconfig is already an `int`

Reply:
Done